generator/osm2meta.cpp

```diff
@@ -148,14 +148,30 @@ string MetadataTagProcessorImpl::ValidateAndFormat_ele(string const & v) const
 
 string MetadataTagProcessorImpl::ValidateAndFormat_destination(string const & v) const
 {
-  return v;
+  {
+   // Normalization. "a1 a2;b1-b2;  c,d "  " -> "a1 a2; b1-b2; c; d".
+   string r;
+   strings::Tokenize(v, ";,", [&](std::string_view d)
+   {
+     if (r.empty())
+       r = strings::Trim(d);
+     else
+       r += "; " + string(strings::Trim(d));
```

Comment:
Please, rebase on fresh master, I changed Trim notation and removed strange returning value.
Should be like this:
```
strings::Trim(d);
if (d.empty())
  return;
if (!r.empty())
  r += "; ";
r.append(d);
```